proxy-attestation-server/CAAttestation.puml

```diff
@@ -0,0 +1,52 @@
+@startuml
```

Comment:
Not sure if there is a better place to store this file

Reply:
Moving it to the wiki